upgradability/set_code_hash/README.md

```diff
@@ -0,0 +1,24 @@
+# Smart contract upgrades with `set_code_hash`
+
+### `set_code_hash` method
+
+When using `set_code_hash` for upgrading contracts we basically tell `pallet-contracts` to assign the code of

```

Comment:
I would link to some docs... the best I found is the first file (docstring from `api.rs`) from https://github.com/paritytech/ink/pull/1203/files